meltdown-mitigation/conditionals.py

```diff
@@ -37,20 +34,32 @@ def reactor_efficiency(voltage, current, theoretical_max_power):
     where generated power = voltage * current
     """
 
-    pass
+    efficiency: Union[int, float] = voltage * current / theoretical_max_power * 100
```

Comment:
Bonus points for specifying a type 😃
But minus bonus points, because `int / int` result is always of `float` type